### PR TITLE
Replace TravisCI with Github Actions

### LINKED ISSUE #62: Implement CI

### PR DESCRIPTION
**Pull request by strawberriesandcheese**
Closes #62 even better
Also encompasses #66

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on: [ push, pull_request, pull_request_review, pull_request_review_comment ]
```
Comment (SilvanVerhoeven) [inline .github/workflows/ci.yml:7]: The comment above says this triggers only on the master branch? Will it also trigger for the dev-branch? Might make sense to trigger it also for all other branches, so stuff doesn't need to be merged into dev before running tests automatically?

File: packages/BaselineOfChangesReloaded.package/BaselineOfChangesReloaded.class/instance/baseline..st [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+baseline: spec
+	<baseline>
+	spec for: #common do: [
+		spec 
+			package: 'ChangesReloaded-Core';
+			package: 'ChangesReloaded-Tests' with: [ spec requires: #('ChangesReloaded-Core') ].
+			
+		spec
+			group: 'default' with: #('ChangesReloaded-Core');
+			group: 'Tests' with: #('ChangesReloaded-Tests')].
```
Comment (SilvanVerhoeven) [inline packages/BaselineOfChangesReloaded.package/BaselineOfChangesReloaded.class/instance/baseline..st:11]:
```suggestion
			group: 'Tests' with: #('ChangesReloaded-Tests')]
```

File: packages/BaselineOfChangesReloaded.package/BaselineOfChangesReloaded.class/instance/baseline..st [hunk 3]
```diff
@@ -0,0 +1,11 @@
+baseline
+baseline: spec
+	<baseline>
```
Comment (SilvanVerhoeven) [inline packages/BaselineOfChangesReloaded.package/BaselineOfChangesReloaded.class/instance/baseline..st:3]:
```suggestion

	<baseline>
```

File: packages/ChangesReloaded-Tests.package/CRBrowserTest.class/instance/changesBrowser..st [hunk 4]
```diff
@@ -0,0 +1,3 @@
+accessing
+changesBrowser: aCRBrowser
+	changesBrowser := aCRBrowser
```
Comment (SilvanVerhoeven) [inline packages/ChangesReloaded-Tests.package/CRBrowserTest.class/instance/changesBrowser..st:3]:
```suggestion

	changesBrowser := aCRBrowser
```

File: packages/ChangesReloaded-Tests.package/CRBrowserTest.class/instance/changesBrowser.st [hunk 5]
```diff
@@ -0,0 +1,3 @@
+accessing
+changesBrowser
+	^ changesBrowser
```
Comment (SilvanVerhoeven) [inline packages/ChangesReloaded-Tests.package/CRBrowserTest.class/instance/changesBrowser.st:3]:
```suggestion

	^ changesBrowser
```

File: .github/workflows/ci.yml (hunk 1 above)
Comment (marcelgarus) [inline .github/workflows/ci.yml:7]: Do `pull_request_review` and `pull_request_review_comment` make sense here? PR reviews and comments don't change anything in the code, so the result of our tests will remain the same (assuming our tests are deterministic which they should be).

> **Note:** Suggestions in reviews or comments need to be manually accepted before they are part of the PR, so even they don't change the code by themselves.

```suggestion
on: [ push, pull_request ]
```

Comment (strawberriesandcheese) [reply .github/workflows/ci.yml:7]: We forgot to edit the comment when we edited this line. It actually triggers for every single branch.
Else we would need to set a property ```branch: [ master ] ``` for each of the triggers.

**Pull request merged**